.github/workflows/publish-maven-central.yml

```diff
@@ -8,12 +8,17 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```

Comment:
why dont we need a checkout action anymore? im particularly worried about this action since you can't test run it afaik